lib/ansible/modules/packaging/os/pkg5_publisher.py

```diff
@@ -197,12 +197,9 @@ def get_publishers(module):
 def unstringify(val):
     if val == "-" or val == '':
         return None
-    elif val == "true":
-        return True
-    elif val == "false":
-        return False
-    else:
-        return val
+    if val in ["true", "false"]:
+        return val == 'true'
+    return val
```

Comment:
I like to leave ```elif``` when it is logical.  You might as well change the logic of the initial condition to match.  I like tuples.  All of that yields:
``` python
if val in ("-", ""):
    return None
elif val in ("true", "false"):
    return val == "true"
return val
```